### PR TITLE
Further localization

### PR DESCRIPTION
**Pull request by Bob-Murphy**
## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Further localization of hard-coded strings plus a few typo fixes here and there

### CODE REVIEW

File: A3A/addons/core/functions/Base/fn_rebuildAssets.sqf [hunk 1]
```diff
@@ -58,7 +58,7 @@ if (isNull _antennaDead) then
 	{
 	_nameX = [_siteX] call A3A_fnc_localizar;
 
-	[_titleStr, format ["%1 Rebuilt"]] call A3A_fnc_customHint; //localize "STR_A3A_fn_base_rebasset_done_1" but what's the %1? 
+	[_titleStr, format ["%1 Rebuilt"]] call A3A_fnc_customHint; //TODO: localize "STR_A3A_fn_base_rebasset_done_1" but what's the %1?
```
Comment (Bob-Murphy) [inline A3A/addons/core/functions/Base/fn_rebuildAssets.sqf:61]: @jaj22

Comment (jaj22) [reply A3A/addons/core/functions/Base/fn_rebuildAssets.sqf:61]: looks bugged, should be `format ["%1 Rebuilt", _nameX]`

File: A3A/addons/core/functions/init/fn_initClient.sqf [hunk 2]
```diff
@@ -386,16 +386,16 @@ fireX allowDamage false;
...
-        "<br/>Time since GC: " + ([[serverTime-A3A_lastGarbageCleanTime] call A3A_fnc_secondsToTimeSpan,1,0,false,2,false,true] call A3A_fnc_timeSpan_format)
+        localize "STR_A3A_gameOpt_title",
+        localize "STR_A3A_version"+" "+ QUOTE(VERSION_FULL) +
+        localize "STR_A3A_resoBal"+" "+ (A3A_enemyBalanceMul / 10 toFixed 1) + "x" +
+        localize "STR_A3A_unlockNo"+" "+ str minWeaps +
+        localize "STR_A3A_limFT"+" "+ (["No","Yes"] select limitedFT) +
+        localize "STR_A3A_spawnDist"+" "+ str distanceSPWN + "m" +
+        localize "STR_A3A_civLim"+" "+ str globalCivilianMax +
+        localize "STR_A3A_timeGC"+" "+ ([[serverTime-A3A_lastGarbageCleanTime] call A3A_fnc_secondsToTimeSpan,1,0,false,2,false,true] call A3A_fnc_timeSpan_format)
     ] call A3A_fnc_customHint;
```
Comment (Bob-Murphy) [inline A3A/addons/core/functions/init/fn_initClient.sqf:399]: I broke this whole block and need a hand with it.
image

File: A3A/addons/core/functions/OrgPlayers/fn_unitTraits.sqf [hunk 3]
```diff
@@ -31,24 +33,24 @@ if(roleDescription player == "Default Commander") then
...
+    	case "I_G_Soldier_AR_F": {player setUnitTrait ["audibleCoef",1.2]; player setUnitTrait ["loadCoef",0.8]; _text = localize "STR_A3A_fn_orgp_unitTraits_autorifle"}; //reintroduced - 8th January 2020, Bob Murphy
+    	case "I_G_engineer_F":  {_text = localize "STR_A3A_fn_orgp_unitTraits_engi"}; //reintroduced - 8th January 2020, Bob Murphy
  };
 };
 
 if (isMultiPlayer) then
 {
 	sleep 5;
-	["Unit Traits", format ["You have selected %1.",_text]] call A3A_fnc_customHint;
+	[_titleStr, format [localize "STR_A3A_fn_orgp_unitTraits_you",_text]] call A3A_fnc_customHint;
```
Comment (Bob-Murphy) [inline A3A/addons/core/functions/OrgPlayers/fn_unitTraits.sqf:55]: I also broke this one.
The `_text` for the different cases is not being filled in.

File: A3A/addons/core/functions/init/fn_initClient.sqf (hunk 2 above)
Comment (jaj22) [reply A3A/addons/core/functions/init/fn_initClient.sqf:399]: The strings just have the wrong names? `STR_A3A_limFT` is `STR_A3A_fn_init_initClient_gameopt_limFT` for example. First one is too short and the second is too long :P

Comment (Bob-Murphy) [reply A3A/addons/core/functions/init/fn_initClient.sqf:399]: Is there a max lenghts for them?

Comment (jaj22) [reply A3A/addons/core/functions/init/fn_initClient.sqf:399]: No, it's just unnecessary.

**Comment by jaj22** [comment]
Hmm, the <br/> doesn't work, does it? XML will eat it unless you do that &gt stuff. Ideally you keep them out of the string table and add them during composition.

File: A3A/addons/core/functions/init/fn_initClient.sqf (hunk 2 above)
Comment (Bob-Murphy) [reply A3A/addons/core/functions/init/fn_initClient.sqf:399]: moving the <br/> did the trick
image

File: A3A/addons/core/functions/OrgPlayers/fn_unitTraits.sqf (hunk 3 above)
Comment (Bob-Murphy) [reply A3A/addons/core/functions/OrgPlayers/fn_unitTraits.sqf:55]: fixed

**Pull request merged**